### ISSUE TITLE
Create dbpedia spotlight unit test using mockup endpoint

### ISSUE THREAD

**Issue by jnehring** [freme-project/Broker#124]
First solve this: https://github.com/freme-project/e-Entity/issues/54

And then have a unit test that exchanges the dbpedia spotlight URL with the mockup endpoint.

Please implement this in a branch and do not merge into master before november 12th

**Comment by josauder** [comment]
Implemented in branch Broker:integration-tests and e-Entity:configurableEndpoint. 

When running `mvn test` from Broker the `application.properties` in `src/test/resources` will be used. This includes the parameter `dbpediaSpotlightEndpointUrl` which in this `application.properties` is set to `localhost:8080/mockups/sparql`.

When running the Broker with any other `application.properties` then this value is not set, and will default to `http://spotlight.nlp2rdf.aksw.org/spotlight`. See https://github.com/freme-project/e-Entity/commit/80416c12405f0f986f67e371cd065793cd4eec22 on how this is implenented in the e-entity repo. This is ready to be implemented but we wait until November 12

**Comment by ArneBinder** [comment]
The e-entity `configurableEndpoint` branch includes the broker to get the enpoint url parameter from `test/**/application.properties`. I think, we should avoid this. @jnehring where and how should we save the mockup url parameter for e-entity?

**Comment by jnehring** [comment]
You should set a default parameter in e-Entity. http://www.mkyong.com/spring3/spring-value-default-value/

So this default parameter gets overriden by the brokers src/test/resources/application.properties file. In every other configuration we use the default value.

Ok?

**Comment by josauder** [comment]
This works fine without the dependency.
I think Ihad just forgotten to take it out of the pom.xml, I don't think it was used anywhere.

I updated the `e-entity/configurableEndpoint` branch, now also FremeNER-Enpoint is condfigurable.
The parameters are configured in:
`src/test/resources/application.properties` file in:
* `freme.eentity.dbpediaSpotlightEndpointUrl=http://localhost:8000/mockups/sparql`
* `freme.eentity.fremeNerEndpointUrl=http://localhost:8000/mockups/sparql`

**Comment by ArneBinder** [comment]
merged into master & deleted `Broker:integration-tests` and `e-Entity:configurableEndpoint`.
